torch/_inductor/triton_heuristics.py

```diff
@@ -723,6 +754,7 @@ def pointwise(size_hints, meta, tile_hint=None, filename=None):
                 [
                     triton_config(size_hints, bs, num_elements_per_warp=256),
                     triton_config(size_hints, bs // 2, num_elements_per_warp=64),
+                    *hinted_configs,
```

Comment:
afaik x*y*z == number of elements, and then we divide by num_elements_per_warp to get the number of warps.

when x -> x/2 and num_elements_per_warp -> num_elements_per_warp/2, then number of warps doesn't change.

In this case I put `bs // 4` because there appears to be a max number of warps of 8 in the triton_config implementation; so we need x*y*z <= 256 in order to actually get num_elements_per_warp=32.

Reply:
It's a bit confusing. add comment ?

Reply:
Added a comment in `def triton_config(...`